docs/GettingStarted/configuration/PelorusCore.md

```diff
@@ -0,0 +1,222 @@
+# Configuration
+
+The Pelorus Core configuration applies to Prometheus, Grafana, Thanos and other operational aspects of the Pelorus stack.
+
+Those configuration options are used in the Pelorus configuration object YAML file to create Pelorus application instance.
+
+Each Pelorus Core configuration option must be placed under `spec` in the Pelorus configuration object YAML file as in the example:
+
+```yaml
+kind: Pelorus
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+metadata:
+  name: pelorus-pelorus-instance
+  namespace: pelorus
+spec:
+  exporters:
+    [...] # Pelorus exporters configuration options
+  # Pelorus Core configuration options
+```
+
+For configuration options for exporters, check [its configuration guide](PelorusExporters.md).
+
+## Example
+
+Configuration part of the Pelorus object YAML file, with some non-default options:
+
+```yaml
+kind: Pelorus
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+metadata:
+  name: pelorus-instance
+  namespace: pelorus
+spec:
+  exporters:
+    [...] # Pelorus exporters configuration options
+  # Below are non-default options
+  openshift_prometheus_basic_auth_pass: mysecretpassword
+  openshift_prometheus_htpasswd_auth: 'internal:{SHA}CM2SM2eJAAllfquBJ1M3m9syHus='
+  prometheus_retention: 500d
+  prometheus_retention_size: 2GB
+  prometheus_storage: true
+  prometheus_storage_pvc_capacity: 3Gi
+  prometheus_storage_pvc_storageclass: mystorageclass
+```
+
+## List of all configuration options
+
+| Variable | Required | Default Value |
```

Comment:
I would change here for a list instead of a table. I think it is not as beautiful, but it will help us when we need to update, to just update info in one place

Reply:
As discussed on slack, nice idea but will stick to table.